alwayson/controllers/injectors.py

```diff
@@ -0,0 +1,9 @@
+from alwayson.config import Config
```

Comment:
Nice file name! Is this _fueling_ the injectors of testerozza? ;) btw, the names grown on me - def a fan

Reply:
Yea I imagine this file contains all of the methods used for dependency injection into the request handlers.